dataload/import_to_elasticsearch.py

```diff
@@ -335,6 +337,13 @@ def update_doc_with_region(grant):
                 grant['recipientDistrictName'] = location["name"]
 
 
+def update_doc_with_title_and_description(grant):
+    """
+    Update ElasticSearch with the new key 'title_and_description'.
+    """
+    grant['title_and_description'] = json.dumps([grant.get('title'), grant.get('description')])
```

Comment:
Why would you do it like that in this case?

Reply:
In the JSON way elasticsearch we would be putting a JSON string i.e `["mytitle", "mydescription"]` into that field and that is a fairly odd thing to do.  Elasicsearch does not know what to do with a JSON string and just try and ignore the non standard characters i.e the `[",]`, so hopefully it will end up treating it like `mytitle mydescription` in the end.  However, if the tile  or description has quotes or unicode in it, then there is a chance elastic search may treat those strings wrongly as it will get an will be json encoding of them.

Reply:
Thanks for the explanation @kindly . I took the idea of doing it like this from line 357-366 
`org["id_and_name"] = json.dumps([found_name, org_id])`
Would it be ok in this case? This also goes to the elastic search, right?

Reply:
The idea behind `id_and_name` was as to make a string that would be used as an identifier for the facets (aggregations) to work. We where having issues where `name` could not be used as the key to the aggregation as some ids had slightly different names associated with it.  So the id_and_name just gets the first name for an id and uses that in the facet search on the left. So `id_and_name` string would be treated as a whole and not be broken up into separate terms by elastic search (it says not_analyzed for that reason in the definition). 

My worry is that say you have Esmée Fairbairn Foundation in the name or definition file somewhere (ie some unicode non ascii) then you get:

```python
>>> json.dumps("Esmée Fairbairn Foundation")
'"Esm\\u00e9e Fairbairn Foundation"'
```
and elastic-search will not know what todo with the `\\u000e9e` bit and the search for Esmée Fairbairn Foundation will fail.